content/en/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -14,9 +14,10 @@ This page provides a real world example of how to configure Redis using a Config
 
 {{% capture objectives %}}
 
-* Create a ConfigMap.
-* Create a pod specification using the ConfigMap.
-* Create the pod.
+* Create a kustomization.yaml with
+  * a ConfigMap generator
+  * a Pod resource config using the ConfigMap
+* Apply the directory by `kubectl apply -k ./`
 * Verify that the configuration was correctly applied.
```

Comment:
Possible change: Apply the directory by calling [running] ...

Reply:
updated